Tests/ApolloCodegenTests/CodeGeneration/Templates/ObjectTemplateTests.swift

```diff
@@ -130,6 +132,21 @@ class ObjectTemplateTests: XCTestCase {
     // then
     expect(actual).to(equalLineByLine(expected, atLine: 3, ignoringExtraLines: true))
   }
+  
+  func test_render_givenKeyFields_rendersKeyFieldArray() {
```

Comment:
Can we add a test for generating the array when you have more than one key field please!